src/app/spans.rs

```diff
@@ -66,11 +85,14 @@ impl SpanConverter {
             kind: self.kind,
             start_time: Some(span.start.into()),
             end_time: Some(span.end.into()),
-            attributes: None, // TODO: attributes
+            attributes: Some(oc::span::Attributes {
+                attribute_map: attributes,
+                dropped_attributes_count: 0,
```

Comment:
nit: prefer `oc::span::Attributes::default()..` (as it's a bit more future-proof if protobuf grows fields